wip-tec/en/WebServicesAccess.md

```diff
@@ -27,9 +41,27 @@ When access to collaboration tools and sites is restricted, instead of increasin
 Users will find a way around the blocks if it makes their life easier.
 Security is more than just locking things down; user experience must also be considered.
 
+#### Bandwidth
+
+The R&D team has been piloting a to route traffic to site that use a lot of bandwidth directly to the Internet.
+This type of solution could mitigate the risk of making network performance worse and could even reduce the stress on the corporate network.
+It would also eliminate the need to block sites that are flagged for using too much bandwidth.
+
+#### Digital Worker Profiles
+
+If not all groups currently require or want more open access to the Internet, we can expect that more and more of them will in the future.
+Current branches that would benefit:
+
+- IITB
+- ...
+
+#### Awareness
```

Comment:
Add another sub category: Perimeter Protection

Explore and evaluate the use of an electronic communication labelling tool. A tool that forces all electronic content to be categorized by classification and can clearly alerts the end user of such classification.  Such tool is to augment the DLP program by providing it with metadata description of the electronic content where, should the content being uploaded does not match the security of the site, it flags infractions for later investigation or just blocks them entirely.